File: Client/src/app/common/Enums/Enums.ts
export enum ErrorCode {
    INTERNAL_SERVER_ERROR = 500,
    BAD_REQUEST = 400,
    NOT_FOUND = 404,
    UNAUTHENTICATE = 401,
    FORBIDDEN = 403,
    TOKEN_EXPIRED = 402,
    SESSION_EXPIRED = 419,
    MISSING_TOKEN = 417,
    INVALID_TOKEN_FORMAT = 406
}


export enum UserStatus {
    Active = 1,
    Inactive = 2,
    Block = 4,
    // Delete = 9
}

export enum AccessRights {
    Admin = 1,
    User = 2
}

export enum Action {
    Insert = 1,
    Update,
    Delete,
    View,
    Other
}
export enum RequisitionStatus {
    Submitted = 1,
    Approved = 2,
    Rejected = 3,
    Closed = 4,
    Cancel=5
}

File: NybSys.Common/Enums/Enums.cs
using System.ComponentModel;

namespace NybSys.Common.Enums
{
    public class Enums
    {
        public enum Status
        {
            Active = 1,
            Inactive = 2,
            Deleted = 9,
            Block = 4
        }
        public enum RequisitionStatus
        {
            Submitted = 1,
            Approved = 2,
            Rejected = 3,
            Closed = 4,
            Cancel = 5

        }
        public enum TokenAlgorithm
        {
            [Description("HS256")]
            HmacSha256,
            [Description("HS384")]
            HmacSha384,
            [Description("HS512")]
            HmacSha512,
            [Description("RS256")]
            RsaSha256,
            [Description("RS384")]
            RsaSha384,
            [Description("RS512")]
            RsaSha512,
            [Description("ES256")]
            EcdsaSha256,
            [Description("ES384")]
            EcdsaSha384,
            [Description("ES512")]
            EcdsaSha512,
            [Description("PS256")]
            RsaSsaPssSha256,
            [Description("PS384")]
            RsaSsaPssSha384
        }
    }

    public enum Action
    {
        Insert=1,
        Update,
        Delete,
        View,
        Other
    }

    public enum Module
    {
        Web=1,
        Desktop,
        Mobile
    }

    public enum LogType
    {
        SecurityLog=1,
        ErrorLog,
        SystemLog,
        DbQuery,
        Other
    }
}
